### ISSUE TITLE
Error: unknown option "-d_z"

### ISSUE THREAD

**Issue by marielohcs** [xqwen/dap#18]
Hi!

I ran make on the dap_src folder and obtained the dap-g executable.

However, when I ran the following code 
dap-g -d_z sample_data/sim.1.zval.dat -d_ld sample_data/sim.1.LD.dat

I got the error of 
Error: unknown option "-d_z"

Please advise; Thanks!

Cheers,
Marie

**Comment by xqwen** [comment]
Hi Marie,

I am unable to replicate your reported error with a complete clone and compile procedure.  Did you directly compile from the source using ```make```? Other possibility may be related to your terminal encoding.

**Comment by marielohcs** [comment]
Dear Xiaoquan

Thanks for the prompt reply! 

Yes, that's what I did (download and compile).
One difference is that for it to work on our cluster, I loaded gsl module and also therefore had to append -I/apps/gsl/2.1/include -L/apps/gsl/2.1/lib to the compiler (i.e after the lines that start with "g++ ..." in the Makefile)..
I don't think that is the problem though?

Also, is it a must to use OpenMP compiler? 

P.S. I will also check with our HPC if they can help...
